modules/system/lang/it/lang.php

```diff
@@ -1,23 +1,61 @@
-<?php
-
-return [
+<?php return [
     'app' => [
         'name' => 'Winter CMS',
         'tagline' => 'Tornare alle origini',
     ],
+    'locale' => [
+        'ar' => 'العربية',
+        'be' => 'Беларуская',
+        'bg' => 'Български',
+        'ca' => 'Català',
+        'cs' => 'Čeština',
+        'da' => 'Dansk',
+        'en' => 'English (United States)',
+        'en-au' => 'English (Australia)',
+        'en-ca' => 'English (Canada)',
+        'en-gb' => 'English (United Kingdom)',
+        'et' => 'Eesti',
+        'de' => 'Deutsch',
+        'el' => 'Ελληνικά',
+        'es' => 'Español',
+        'es-ar' => 'Español (Argentina)',
+        'fa' => 'فارسی',
+        'fr' => 'Français',
+        'fr-ca' => 'Français (Canada)',
+        'hu' => 'Magyar',
+        'id' => 'Bahasa Indonesia',
+        'it' => 'Italiano',
+        'ja' => '日本語',
+        'kr' => '한국어',
+        'lt' => 'Lietuvių',
+        'lv' => 'Latviešu',
+        'nb-no' => 'Norsk (Bokmål)',
+        'nl' => 'Nederlands',
+        'pl' => 'Polski',
+        'pt-br' => 'Português (Brasil)',
+        'pt-pt' => 'Português (Portugal)',
+        'ro' => 'Română',
+        'rs' => 'Srpski',
+        'ru' => 'Русский',
+        'fi' => 'Suomi',
+        'sv' => 'Svenska',
+        'sk' => 'Slovenský',
+        'sl' => 'Slovenščina',
+        'th' => 'ไทย',
+        'tr' => 'Türkçe',
+        'uk' => 'Українська мова',
+        'zh-cn' => '简体中文',
+        'zh-tw' => '繁體中文',
+        'vn' => 'Tiếng việt',
+    ],
     'directory' => [
         'create_fail' => 'Impossibile creare la cartella: :name',
     ],
     'file' => [
         'create_fail' => 'Impossibile creare il file: :name',
     ],
-    'page' => [
-        'invalid_token' => [
-            'label' => 'Token di protezione non valido',
-        ],
-    ],
     'combiner' => [
-        'not_found' => "Il file combinatore ':name' non è stato trovato.",
+        'not_found' => 'Il file combinatore \':name\' non è stato trovato.',
```

Comment:
Try changing external single quotes to double quotes when you have single quotes inside instead of escaping the single quotes.